client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx

```diff
@@ -0,0 +1,209 @@
+import classNames from 'classnames';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import FoldableCard from 'components/foldable-card';
+import SectionHeader from 'components/section-header';
+import { checkInboundTransferStatus } from 'lib/domains';
+import support from 'lib/url/support';
+
+class TransferDomainPrecheck extends React.PureComponent {
+	static propTypes = {
+		domain: PropTypes.string,
+		setValid: PropTypes.func,
+	};
+
+	state = {
+		unlocked: false,
+		privacy: false,
+		email: '',
+		loading: true,
+	};
+
+	componentWillMount() {
+		this.refreshStatus();
+	}
+
+	componentWillUpdate( nextProps ) {
+		if ( nextProps.domain !== this.props.domain ) {
+			this.refreshStatus();
+		}
+	}
+
+	onClick = () => {
+		this.props.setValid( this.props.domain );
+	};
+
+	refreshStatus = () => {
+		this.setState( { loading: true } );
+
+		checkInboundTransferStatus( this.props.domain, ( error, result ) => {
+			if ( ! isEmpty( error ) ) {
+				return;
+			}
+
+			this.setState( {
+				email: result.admin_email,
+				privacy: result.privacy,
+				unlocked: result.unlocked,
+				loading: false,
+			} );
+		} );
+	};
+
+	getSection( heading, message, explanation, position, button ) {
+		const header = (
+			<div>
+				<div className="transfer-domain-step__section-heading">
+					<span className="transfer-domain-step__section-heading-number">{ position }</span>
+					{ heading }
+				</div>
+				<div>
+					<small>{ message }</small>
+				</div>
+			</div>
+		);
+
+		const cardClasses = classNames( {
+			'hide-secondary': ! button,
+		} );
+
+		return (
+			<FoldableCard
+				className={ cardClasses }
+				header={ header }
+				summary={ button }
+				expandedSummary={ button }
+			>
+				{ explanation }
+			</FoldableCard>
+		);
+	}
+
+	getStatusMessage() {
+		const { translate } = this.props;
+		const { unlocked } = this.state;
+
+		const heading = unlocked
+			? translate( 'Domain is unlocked.' )
+			: translate( 'Unlock the domain.' );
+		const message = unlocked
+			? translate( 'Your domain is unlocked at your current registrar.' )
+			: translate(
+					"You'll need to unlock the domain at your current registrar before we can move it."
+				);
+		const explanation = translate(
+			'Your current domain provider has locked the domain to prevent unauthorized transfers.'
+		);
+
+		const button = unlocked ? (
+			<div className="transfer-domain-step__unlocked">
+				<Gridicon icon="checkmark" size={ 12 } />
+				<span>{ translate( 'Unlocked' ) }</span>
+			</div>
+		) : (
+			<Button disabled={ this.state.loading } onClick={ this.refreshStatus }>
+				{ translate( 'Check Again' ) }
+			</Button>
+		);
+
+		return this.getSection( heading, message, explanation, 1, button );
+	}
+
+	getPrivacyMessage() {
+		const { translate } = this.props;
+		const { email, privacy } = this.state;
+
+		const heading = privacy
+			? translate( 'Disable privacy protection.' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'Privacy Protection' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/style-transfer-precheck-screen)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).